src/lazy/text/buffer.rs

```diff
@@ -2660,8 +2668,8 @@ mod tests {
     }
 
     macro_rules! matcher_tests_with_macro {
-        ($parser:ident $macro_src:literal $($expect:ident: [$($input:literal),+$(,)?]),+$(,)?) => {
-            mod $parser {
+        ($mod_name:ident $parser:ident $macro_src:literal $($expect:ident: [$($input:literal),+$(,)?]),+$(,)?) => {
+            mod $mod_name {
```

Comment:
🗺️ This macro creates a module with a bunch of nested unit tests. However, the module name was also the name of the parser. This meant you couldn't write two sets of unit tests that used the same parser entry point. I added another parameter (`$mod_name`) to this macro to allow the module name to be specified separately.